01-front-end-basics/jquery-dom-manipulation/24/README.md

```diff
@@ -0,0 +1,252 @@
+```
+
+### Including jQuery
+
+To use jQuery, you must include the library in your page. The quickest way to include jQuery in your project is to copy the link to the CDN (content delivery network) and put it into a script tag at the bottom of your HTML `<body>`. You can get the CDN by searching for "jQuery" on <a href="https://cdnjs.com" target="_blank">cdnjs</a>.
+
+**Important:** Your JavaScript file MUST come after jQuery in the order you require scripts. This is because the page loads in order, so the jQuery library must be loaded before you can use any of it's functionality.
+
+``` html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+</head>
+<body>
+  <!-- html code here -->
+
+  <!-- jquery -->
+  <script type="text/javascript" src="https://code.jquery.com/jquery-1.11.3.min.js"></script>
+
+  <!-- custom script -->
+  <script type="text/javascript" src="main.js"></script>
+</body>
+</html>
+```
+
+Sites like GitHub, CSS-Tricks, and jquery.com (!) all include the jQuery library on their page. This means all you have to do is open up the Chrome Developer Console on one of those sites, and you can start experimenting with jQuery on the page.
+
+### Using jQuery
+
+You know you're working with jQuery any time you see a `$` in your code.
+
+As you're working with jQuery to manipulate DOM elements, you are almost always either *getting* or *setting* a value. Memorize this pattern:
+
+* `$('css-selector').someJqueryMethodName()` - **get** a value
+* `$('css-selector').someJqueryMethodName(setterValue)` - **set** a value
+
+For example, if you are viewing this `readme` on GitHub, you can try the following in your Chrome Developer Console:
+
+* `$('article').text()` - **get** the `readme`'s text (it lives inside of an `article` tag)
+* `$('article').text('Boo!')` - **set** the `readme`'s text to "Boo!"
+
+Let's try another:
+
+* `$('article').css('background-color')` - **get** the background color
+* `$('article').css('background-color', 'blue')` - **set** the background color to blue
+
+### Documentation
+
+Check out the `.text()` and `.css()` methods in the jQuery API Documentation: <a href="http://api.jquery.com/text" target="_blank">text</a>, <a href="http://api.jquery.com/css" target="_blank">css</a>
+
+* Pay close attention in the documentation - there's one section that talks about how to *get* text, and there's another section that talks about how to *set* text.
+* `.text()` and `.css()` are not native javascript methods! They only work on jQuery-wrapped DOM Elements (that's what that `$` is doing).
+
+For more cool DOM manipulation tricks, you'll need to hit the docs:
+
+* <a href="http://api.jquery.com" target="_blank">jQuery API Documentation</a>
+  * <a href="http://api.jquery.com/category/manipulation" target="_blank">DOM Manipulation Methods</a>
+  * <a href="http://api.jquery.com/category/traversing" target="_blank">Tree Traversal Methods</a>
+
+## Challenges
+
+### Draw a Tree
```

Comment:
The challenges is the first time "Tree" is mentioned.  Perhaps it should be discussed above.